app.py

```diff
@@ -289,3 +311,9 @@ def page_not_found(error):
 def logout():
     session.pop('user')
     return redirect('/signin')
+
+
+# TODO: Remove this temporary test function
+@app.route('/testscratchcard')
+def testscratchcard():
+    return render_template('scratch-card.html')
```

Comment:
Remove if needed

Reply:
Thanks for catching that! I'll remove it.